ui/src/views/Users.vue

```diff
@@ -124,6 +124,7 @@ import SearchBar from "@/components/SearchBar.vue";
 import Table from "@/components/Table.vue";
 import InlineRowEdit from "@/components/InlineRowEdit.vue";
 import FeedbackMessage from "@/components/FeedbackMessage.vue";
+
 import { deleteUser, getUsers, putUser, getProjects } from "@/api/api";
```

Comment:
please remove whitespace

Reply:
Removed

Reply:
I keep it as it separates libs from components